lib/gui/app.js

```diff
@@ -123,10 +125,10 @@ app.run((AnalyticsService, ErrorService, UpdateNotifierService, SelectionStateMo
 
     AnalyticsService.logEvent('Notifying update');
 
-    return UpdateNotifierService.notify();
-
+    return s3.getLatestVersion(s3.BUCKET_URL_PRODUCTION, {
```

Comment:
> Which of course results in two separate S3 calls, and seems a bit inefficient to me.

Remember that the first S3 call is cached, so we never make more than one HTTP request. I'm happy to refactor if this helps to clarify things.

> Why not just have a single call to getLatestVersion() at the top here, get rid of isLatestVersion, and compare the version here directly??

Sounds good to me.